dev/com.ibm.ws.clientcontainer.8.0_fat/fat/src/com/ibm/ws/clientcontainer/fat/AppClientTest.java

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ * Copyright (c) 2015 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.clientcontainer.fat;
+
+import org.junit.Test;
+
+import com.ibm.websphere.simplicity.ProgramOutput;
+
+public class AppClientTest extends AbstractTest {
+
+    /*
+     * Basic client launch test.
+     * e.g., "client run com.ibm.ws.clientcontainer.fat.ClientContainerClient"
+     * Check if the test application is printing out "Hello Application Client." to the console.
+     */
+    @Test
+    public void testHelloAppClient() throws Exception {
+        client.copyFileToLibertyClientRoot("clients/HelloAppClient/client.xml");
```

Comment:
Is it really necessary to copy the client.xml in the beginning of each test case?  This should be done automatically by the FAT framework already.